.github/workflows/integration-test-upgrade-suite.yaml

```diff
@@ -94,12 +94,11 @@ jobs:
 
     - name: Artifact
       uses: actions/upload-artifact@v2
-      if: failure()
       with:
         name: ceph-upgrade-suite-artifact-${{ matrix.kubernetes-versions }}
         path: /home/runner/work/rook/rook/tests/integration/_output/tests/
 
     - name: setup tmate session for debugging when event is PR
-      if: failure() && github.event_name == 'pull_request'
+      if: github.event_name == 'pull_request'
```

Comment:
Was this only for temporary testing and you'll revert this commit?

Reply:
Yes. I'll delete the commit when the output looks good.